dm/dm/ctl/master/query_validation.go

```diff
@@ -81,36 +79,36 @@ func queryValidationError(cmd *cobra.Command, _ []string) (err error) {
 
 func NewQueryValidationStatusCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "status <task-name>",
+		Use:   "status [--stage stage] <task-name>",
 		Short: "query validation status of a task",
 		RunE:  queryValidationStatus,
 	}
-	cmd.Flags().String("stage", "running", "filter status")
+	cmd.Flags().String("stage", "", "filter validation tasks status by stages: Running/Stopped")
```

Comment:
why default value removed?

`Running/Stopped` is hard to type, all lower case is better.

Reply:
> why default value removed?

The default value `""` means showing both `Running` and `Stopped` validation tasks. If we set the default value as `running` then this command couldn't show all tasks' but only either `Running` or `Stopped` tasks' statuses.